src/extension.js

```diff
@@ -97,3 +136,13 @@ supportedLanguages.forEach((language) => {
     },
   });
 });
+
+waitForActiveSupportedDocument()
+  .then((document) => warmUpWorker(document))
+  .then(() => {
+    outputChannel.appendLine('Worker has been warmed up');
+  })
+  .catch((error) => {
+      console.error('Could not warm up worker with first available document:', error);
```

Comment:
I'm not familiar with VSCode API, but how is this different from `outputChannel`? Can we only use one kind of logs?

Reply:
An output channel is shown in the Ui under the "OUTPUT" section on the bottom panel. AFAIK its a simple text buffer, with no formatting options, so it is more of an information to the user rather than a place to log stacktraces.
![](https://github.com/idahogurl/vs-code-prettier-eslint/assets/6181929/614c5951-2270-42df-b7e8-15e0b29df229)

The `console.error` is only shown to developers while debugging the extension. It is here because the output of rejected Promises aren't logged to the console (or at least they shouldn't according to the spec).

Reply:
@netux Let's stick to the `outputChannel` because that will help the user provide us with the error and stack trace in case of debugging.